templates/common/backends.tmpl

```diff
@@ -0,0 +1,34 @@
+{{ define "backend" }}
+  backend "{{ .Kind }}" {
+  {{ if eq .Kind "s3" }}
+    bucket         = "{{ .S3.Bucket }}"
+    {{ if .S3.DynamoTable }}dynamodb_table = "{{ .S3.DynamoTable }}"{{ end }}
+    key            = "{{ .S3.KeyPath }}"
+    encrypt = true
+    region = "{{ .S3.Region }}"
+    profile = "{{ .S3.Profile }}"
+  {{ else if eq .Kind "remote" }}
+    hostname = "{{ .Remote.HostName }}"
+    organization = "{{ .Remote.Organization }}"
+    workspaces {
+      name = "{{ .Remote.Workspace }}"
+    }
+  {{ end }}
+  }
+{{ end }}
+
+{{ define "remote_backend" }}
+  {{ if eq .Kind "s3" }}
+    bucket         = "{{ .S3.Bucket }}"
+    {{ if .S3.DynamoTable }}dynamodb_table = "{{ .S3.DynamoTable }}"{{ end }}
+    key            = "{{ .S3.KeyPath }}"
```

Comment:
probably missing something here but backend and remote_backend look exactly the same except for encrypt = true. Do we need the 2 definitions?

Reply:
Yeah in one case we need `workspace = {` and the other `workspaces {`. :(

Reply:
:sob: